npm-js/create-akkasls-entity/template/base/package.json

```diff
@@ -7,11 +7,7 @@
     "npm": ">=6.0.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.3.0",
-    "@grpc/proto-loader": "^0.1.0",
-    "@lightbend/akkaserverless-javascript-sdk": "0.7.0-beta.11",
-    "google-protobuf": "^3.0.0",
-    "protobufjs": "^6.11.2"
+    "@lightbend/akkaserverless-javascript-sdk": "0.7.0-beta.13"
```

Comment:
Side note: we probably want this version to be updated automatically at some point...

Reply:
yes, that work is not finalized yet. Going to work on it.